### PR TITLE
Invoice

### PR DESCRIPTION
**Pull request by nicodeheza**
- Endpoint to be called by the cron job runner. (send email and create invoice record)
- Endpoint to download invoice PDF

issue: #16

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -13,10 +13,33 @@ model Organization {
...
   createdAt DateTime  @default(now())
   updatedAt DateTime  @default(now()) @updatedAt
 }
 
+enum SubscriptionStatus {
+  ACTIVE
+  INACTIVE
+}
+
+model invoice{
```
Comment (aviggiano) [inline prisma/schema.prisma:28]: Can you put it in PascalCase, such as `Invoice`?

**Review by aviggiano — APPROVED** [review]
Looks good! I'll test it locally before merging, but I don't have any other comments

**Comment by aviggiano** [comment]
Using https://cron-job.org/en/

**Comment by nicodeheza** [comment]
Yes `request.headers['x-api-key'] `

**Comment by aviggiano** [comment]
Closed in favor of #22

**Pull request closed without merging**